tests/golden/django_patterns.py

```
@@ -0,0 +1,94 @@
+
+# --- DBC-001: happy-path only ---
+def test_create_user_happy():  # expect: PYTEST-DBC-001
+    user = create_user("alice")
+    assert user.name == "alice"
+
+
+# --- Clean test ---
+def test_with_raises():
+    """Given invalid input when creating user then ValueError raised."""
+    with pytest.raises(ValueError):
+        create_user("")
```

Ruff (0.15.12)

[error] Undefined name `create_user`

(F821)